src/main/kotlin/org/jitsi/rtp/rtp/RedPacketParser.kt

```diff
@@ -199,6 +182,5 @@ internal class RtpRedPacket(buffer: ByteArray, offset: Int, length: Int) : RtpPa
         val parser = RedPacketParser { b, o, l -> RtpPacket(b, o, l) }
     }
 
-    fun decapsulate() = parser.decapsulate(this)
-    fun parseRedundancyAndDecapsulate(): List<RtpPacket> = parser.parseRedundancyAndDecapsulate(this)
+    fun parse(parseRedundancy: Boolean) = parser.parse(this, parseRedundancy)
```

Comment:
I do think the named API methods were nice though (`decapsulate` and `decapsulateWithRedundancy` or something instead of `parse(true)` and `parse(false)`)

Reply:
I like the simple one better

Reply:
'Simple' as in 'shorter'?  Reading `parse(true)` and `parse(false)` don't really tell you what's happening.